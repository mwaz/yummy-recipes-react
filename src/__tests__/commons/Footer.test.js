import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Footer from '../../components/common/Footer';

describe('Footer component', () => {
  const wrapper = shallow(<Footer />);

  it('renders properly without crashing', () => {
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
