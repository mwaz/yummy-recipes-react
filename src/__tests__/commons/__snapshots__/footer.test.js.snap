// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders properly without crashing 1`] = `
<div
  className="footer"
>
  <Navbar
    color="white"
    expand="md"
    light={true}
    tag="nav"
  >
    <Nav
      className="ml-auto"
      navbar={true}
      tag="ul"
      vertical={false}
    >
      <NavItem
        tag="li"
      >
        <NavLink
          tag="a"
        >
          All rights reserved (c) 2018
        </NavLink>
      </NavItem>
    </Nav>
  </Navbar>
</div>
`;
